R/cookies.R

```diff
@@ -124,24 +130,30 @@ cookie_banner_ui <- function(id, name = "DfE R-Shiny dashboard template") {
 #' @export
```

Comment:
We should add the @import line here as well:
`#' import shiny shinyjs`

and re-run `devtools::document()` to update the NAMESPACE file.